lib/perl/Genome/Process.pm

```diff
@@ -227,6 +227,11 @@ sub _schedule_process {
 
 sub create {
     my $class = shift;
+
+    if ($class->__meta__->is_abstract) {
+        return $class->SUPER::create(@_); #UR will re-call with the concrete class.
+    }
+
```

Comment:
I seen it before, and believe it to do the 'right' thing. I dislike it. Your reason is reasonable :octocat: